src/Renderer/AirspaceRendererGL.cpp

```diff
@@ -138,15 +138,15 @@ class AirspaceVisitorRenderer final
 
 private:
   bool SetupOutline(const AbstractAirspace &airspace) {
-    AirspaceClass type = airspace.GetClass();
+    AirspaceClass asclass = airspace.GetType() ? airspace.GetClass() : airspace.GetType();
```

Comment:
Yet another variable renamed, so much noise in this patch!

And please don't cast an enum to a bool. This is fragile.
I don't even know what this ternary expression is supposed to do. This is very obscure.